server/src/main/java/org/apache/uniffle/server/storage/LocalStorageManager.java

```diff
@@ -81,7 +82,8 @@ public class LocalStorageManager extends SingleStorageManager {
     // We must make sure the order of `storageBasePaths` and `localStorages` is same, or some unit test may be fail
     CountDownLatch countDownLatch = new CountDownLatch(storageBasePaths.length);
     AtomicInteger successCount = new AtomicInteger();
-    ExecutorService executorService = Executors.newCachedThreadPool();
+    ExecutorService executorService = Executors.newCachedThreadPool(
```

Comment:
This is a temporary thread pool. I think it's unnecessary to be a daemon thread pool.WDYT?

Reply:
Would it be better to create the thread pool when the `LocalStorageManager` is initialized.

Reply:
We only use this thread pool to delete the files once when the server start. I think it's no need.